docs/governance-workbook/README.md

```diff
@@ -6,6 +6,125 @@
 
 The governance workbook is an Azure Monitor workbook that provides a customizable single pane of glass for governance.
 
+## Workbook Overview
+Take advantage of the Azure governance workbook to easily navigate through the resource of your subscription. The purpose of this tool is to help you to identify the resources deployed into your environment and keep track on them.
+The following section will help you to understand the metrics displayed in this workbook to ease your cloud usage and administration.
+This tool is contain 14 sections:
+-	Overview
+-	Virtual Machine
+-	Storage + backup
+-	Network
+-	PaaS
+-	Security
+-	Monitoring
+-	Services retirement
+-	Resource age
+-	Tag Explorer
+-	Cost management
+-	Usage + limits
+-	Compliance
+-	Governance
+
+:warning: If the number of result for a query is >10000, please reduce the scope (subscription number) of the analysis
```

Comment:
Is there an error that happens? Might be good to mention what the error message is.